packages/global-styles/src/global.d.ts

```diff
@@ -16,8 +16,14 @@ declare module '@wordpress/components' {
 	export const __experimentalVStack: React.ComponentType< Props >;
 }
 
-declare module '@wordpress/edit-site/build-module/components/global-styles/context';
-declare module '@wordpress/edit-site/build-module/components/global-styles/global-styles-provider';
+// Note: edit-site doesn't have anything on DefinitelyTyped yet :(
+declare module '@wordpress/edit-site/build-module/components/global-styles/context' {
+	export const GlobalStylesContext: React.Context< T >;
```

Comment:
I'd like to provide more specific types for this ideally, but I wasn't sure how to import anything from the `./types` file, and declaring them in this file meant I couldn't export them... but this fixes the issue 🤷

Reply:
Will likely remove this after #78711